### PR TITLE
feat: playwright support

### PR DESCRIPTION
**Pull request by maxigimenez**
# Pull Request Template

## Description

Add playwright support 🎉

Kapture 2020-08-05 at 9 54 57

Screenshot 2020-08-05 at 09 55 16

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

## How Has This Been Tested?

- Tested manually
- Added unit tests for key changes

## Checklist:

- [x] My code follows the style guidelines of this project. `npm run lint` passes with no errors.
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes. `npm run test` passes with no errors.

Closing #82

### CODE REVIEW

**Comment by maxigimenez** [comment]
@tnolet ready for review. I will open a follow-up PR after this, with some small refactor on the code generator, to share code between playwright/puppeteer, avoided here to prevent a massive one.

File: src/popup/components/ResultsTab.vue [hunk 1]
```diff
@@ -1,21 +1,63 @@
 <template>
   <div class="tab results-tab">
+    <div class="tabs">
+      <button v-for="tab in tabs" :key="tab" class="tabs__action" v-bind:class="{'selected': activeTab === tab}" @click.prevent="changeTab(tab)">{{ tab }}</button>
```
Comment (tnolet) [inline src/popup/components/ResultsTab.vue:4]: Would be cool if we could fit a little Puppeteer and Playwright icon just before the wording, i.e. "🎭 Playwright". Not sure if that translates well given the space we have in the tab.

Comment (maxigimenez) [reply src/popup/components/ResultsTab.vue:4]: Will give a try

Comment (maxigimenez) [reply src/popup/components/ResultsTab.vue:4]: @tnolet better?
Screenshot 2020-08-05 at 19 58 40

**Pull request merged**